app/addons/documents/layouts.js

```diff
@@ -47,13 +47,16 @@ export const TabsSidebarHeader = ({
   onGlobalModeSelected,
   globalMode
 }) => {
-  const partKeySelector = (<PartitionKeySelectorContainer
-    databaseName={dbName}
-    partitionKey={partitionKey}
-    onPartitionKeySelected={onPartitionKeySelected}
-    onGlobalModeSelected={onGlobalModeSelected}
-    globalMode={globalMode}/>
-  );
+  let partKeySelector = null;
```

Comment:
Not related to the redux refactoring. 
Just a small fix to prevent React warnings that some props are not set.